agent/provider/src/main.rs

```diff
@@ -42,7 +42,6 @@ async fn main() -> anyhow::Result<()> {
         Commands::Preset(presets_cmd) => presets_cmd.run(config),
         Commands::Profile(profile_cmd) => profile_cmd.run(config),
         Commands::ExeUnit(exe_unit_cmd) => exe_unit_cmd.run(config),
-        Commands::Keystore(keystore_cmd) => keystore_cmd.run(config),
```

Comment:
As we discussed: We need cli for adding removing certificates

Reply:
Will be done here: #2134